vexim/config/header.php

```diff
@@ -13,9 +15,9 @@
     print     "-- " . $_SESSION['domain'] . " ";
   }
   if (isset($_SESSION['domain_id'])) {
-    if (($domheaderrow['enabled'] == "0") || ($domheaderrow['enabled'] == "f")) {
+    if (($domheaderrow['enabled'] === "0") || ($domheaderrow['enabled'] === "f")) {
```

Comment:
This "f" was inserted here:
https://github.com/vexim/vexim2/blob/1e807f14b2bd1d9d9cc9b918983e03bcb600c2f6/vexim/config/header.php

Do you see why this was put there?